metadrive/examples/drive_in_single_agent_env.py

```diff
@@ -41,7 +43,7 @@
         config.update(
             dict(
                 image_observation=True,
-                sensors=dict(rgb_camera=(RGBCamera, 512, 256)),
+                sensors=dict(rgb_camera=(InstanceCamera, 512, 256)),
```

Comment:
We can add a new argument to allow users to select what type of camera to use. But by default, it should be RGBCamera.